docs/querying/set-query-context.md

```diff
@@ -0,0 +1,240 @@
+---
+id: set-query-context
+title: "Set query context"
```

Comment:
Suggest a more user-centric title:
Modify query behavior with the query context
Configure query behavior with the query context.